README.md

```diff
@@ -1,12 +1,91 @@
-# Git Repository Template
+# Nimble Fastlane
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+This library allows developers to easily integrate all shared lanes (e.g., building, deploying) into iOS projects.
+
+## Requirements
+
+- Swift 5.7+
 
 ## Usage
 
-Clone the repository
+### Step 1
+
+To use this library (and [fastlane][fastlane]) in your iOS project, you need to create an executable Swift Package project with:
+
+```bash
+swift package init --type executable --name Fastlane
+```
+
+### Step 2
+
+Add the [nimble-fastlane][nimble-fastlane] dependency to your `Package.swift`.
+
+```swift
+.package(url: "https://github.com/nimblehq/nimble-fastlane", branch: "main")
+```
+
+Here is an example:
+
+```swift
+// swift-tools-version: 5.7
+
+import PackageDescription
+
+let package = Package(
+    name: "FastlaneRunner",
+    products: [
+        .executable(
+            name: "fastlaneRunner",
+            targets: ["FastlaneRunner"]
+        )
+    ],
+    dependencies: [
+        .package(url: "https://github.com/nimblehq/nimble-fastlane", branch: "main")
+    ],
+    targets: [
+        .executableTarget(
+            name: "FastlaneRunner",
+            dependencies: [
+                .byNameItem(name: "NimbleFastlane", condition: nil)
+            ],
+            path: "FastlaneRunner"
+        )
+    ]
+)
+```
+
+### Step 3
+
+Add an entry point (`@main`) or a `main.swift` file (mandatory for executable Swift Packages), configure `Constants`, and start the `fastlane runloop`. For example:
+
```

Comment:
With `swift package init --type executable --name FastlaneRunner` at step 1, we had a file `FastlaneRunner.swift` with the following content (that included `@main` already):
```swift
@main
public struct FastlaneRunner {
    public private(set) var text = "Hello, World!"

    public static func main() {
        print(FastlaneRunner().text)
    }
}
```
So I think this step would naturally be like: "Edit the `FastlaneRunner.swift` to import `Fastlane` and `FastlaneTemplates`, then configure `Constants`, and start the `fastlane runloop`. For example:"